plugin/trino-kafka/src/test/java/io/trino/plugin/kafka/schema/confluent/TestAvroSchemaConverter.java

```diff
@@ -326,15 +318,9 @@ public void testEmptyFieldStrategyForEmptySchema()
                 .hasMessage("Struct type has no valid fields for schema: '%s'", SchemaBuilder.record("nested_record").fields().endRecord());
 
         List<Type> typesForAddDummyStrategy = ImmutableList.<Type>builder()
-                .add(RowType.from(ImmutableList.<RowType.Field>builder()
-                        .add(new RowType.Field(Optional.of(DUMMY_FIELD_NAME), BOOLEAN))
-                        .build()))
-                .add(new ArrayType(RowType.from(ImmutableList.<RowType.Field>builder()
-                        .add(new RowType.Field(Optional.of(DUMMY_FIELD_NAME), BOOLEAN))
-                        .build())))
-                .add(createType(RowType.from(ImmutableList.<RowType.Field>builder()
-                        .add(new RowType.Field(Optional.of(DUMMY_FIELD_NAME), BOOLEAN))
-                        .build())))
+                .add(RowType.from(ImmutableList.of(new RowType.Field(Optional.of(DUMMY_FIELD_NAME), BOOLEAN))))
+                .add(new ArrayType(RowType.from(ImmutableList.of(new RowType.Field(Optional.of(DUMMY_FIELD_NAME), BOOLEAN)))))
+                .add(createType(RowType.from(ImmutableList.of(new RowType.Field(Optional.of(DUMMY_FIELD_NAME), BOOLEAN)))))
```

Comment:
Doesn't belong to `Remove unnecessary Immutable(List|Set|Map) builder use`, as it doesn't remove builder use